src/components/CreateCoursePage/index.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { Icon, StatusAlert } from '@edx/paragon';
+import { push } from 'connected-react-router';
+
+import CreateCourseForm from './CreateCourseForm';
+import store from '../../data/store';
+
+class CreateCoursePage extends React.Component {
+  componentDidMount() {
+    this.props.fetchOrganizations();
+  }
+
+  handleCourseCreate(options) {
+    const courseData = {
+      org: options.courseOrg,
+      title: options.courseTitle,
+      number: options.courseNumber,
+      mode: options.courseEnrollmentTrack,
+      price: options.coursePrice,
+    };
+    this.props.createCourse(courseData);
+  }
+
+  showErrorStatus(error) {
+    return (
+      <StatusAlert
+        id="error"
+        alertType="danger"
+        dismissible={false}
+        open
+        dialog={error}
+        className={['text-center', 'mx-auto', 'w-50']}
+      />
```

Comment:
We might want to pull this specific style of status alert into a simple component that just takes a string.

Reply:
I am using the one @jlajoie already added now.